packages/preview-middleware-client/src/cpe/changes/service.ts

```diff
@@ -176,13 +204,7 @@ export class ChangeService {
                 .filter((change) => !!change) as SavedPropertyChange[]
         ).sort((a, b) => b.timestamp - a.timestamp);
         this.savedChanges = changes;
-        sendAction(
-            changeStackModified({
-                saved: changes,
-                pending: []
-            })
-        );
-        this.options.rta.attachUndoRedoStackModified(this.createOnStackChangeHandler(sendAction));
+        return changes;
```

Comment:
This `return` seems to be not needed. Please remove

Reply:
Changed in: [6a06cae](https://github.com/SAP/open-ux-tools/pull/1443/commits/6a06caec676f9b443e88a72ec972c2d0d85efd66)

Reply:
Please also fix the JS Doc (`@returns`).

Reply:
Changed in: [99858e5](https://github.com/SAP/open-ux-tools/pull/1443/commits/99858e570344d0b785ab50e8a9d6d1d6ebb9f801)